### PR TITLE
etcdmain: configurable 'etcd' binary log-output

### PR DESCRIPTION
**Pull request by gyuho**
@heyitsanthony @xiang90 

Fix https://github.com/coreos/etcd/issues/5449

### CODE REVIEW

File: etcdmain/help.go [hunk 1]
```diff
@@ -139,6 +139,8 @@ logging flags
 		enable debug-level logging for etcd.
 	--log-package-levels ''
 		specify a particular log level for each etcd package (eg: 'etcdmain=CRITICAL,etcdserver=DEBUG').
+	--log-output ''
+		specify 'stdout' or 'stderr' to skip journald logging even when running under init (ppid == 1).
```
Comment (xiang90) [inline etcdmain/help.go]: even when running under systemd

File: etcdmain/config.go [hunk 2]
```diff
@@ -184,6 +184,7 @@ func newConfig() *config {
 	// logging
 	fs.BoolVar(&cfg.Debug, "debug", false, "Enable debug-level logging for etcd.")
 	fs.StringVar(&cfg.LogPkgLevels, "log-package-levels", "", "Specify a particular log level for each etcd package (eg: 'etcdmain=CRITICAL,etcdserver=DEBUG').")
+	fs.StringVar(&cfg.LogOutput, "log-output", "", "Specify 'stdout' or 'stderr' to skip journald logging even when running under init (ppid == 1).")
```
Comment (xiang90) [inline etcdmain/config.go]: we do not need to add this to cfg. the main pkg takes care of it anyway.

**Comment by xiang90** [comment]
LGTM. defer to @heyitsanthony

File: etcdmain/config.go [hunk 3]
```diff
@@ -184,6 +185,7 @@ func newConfig() *config {
 	// logging
 	fs.BoolVar(&cfg.Debug, "debug", false, "Enable debug-level logging for etcd.")
 	fs.StringVar(&cfg.LogPkgLevels, "log-package-levels", "", "Specify a particular log level for each etcd package (eg: 'etcdmain=CRITICAL,etcdserver=DEBUG').")
+	fs.StringVar(&cfg.logOutput, "log-output", "", "Specify 'stdout' or 'stderr' to skip journald logging even when running under systemd.")
```
Comment (heyitsanthony) [inline etcdmain/config.go]: The systemd/journald stuff seems forced, maybe:
`fs.StringVar(&cfg.logOutput, "log-output", "default", "Specify a logging target (stdout,stderr,default).")`

**Comment by gyuho** [comment]
@heyitsanthony @xiang90 All fixed.

**Comment by xiang90** [comment]
LGTM

**Pull request merged**